docs/sources/developer/Developing-Ubuntu.md

```diff
@@ -0,0 +1,120 @@
+# Developing on Ubuntu
+
+## Git
+To download the latest version of Plaso you'll need to install the git tools:
+```
+sudo apt-get install git
+```
+
+Checkout the Plaso source from the git repo:
+```
+git clone https://github.com/log2timeline/plaso.git
+```
+
+## Dependencies
+
+Install the dependencies, using either:
+* A [virtualenv](Developing-Virtualenv.html#Fedora%20Core)
+* The [pre-packaged dependencies](Dependencies.html#Fedora%20Core).
+
+Check if you have all the dependencies installed and have the right minimum 
+versions:
+```
+python utils/check_dependencies.py
+```
+
+**Note that some dependencies are actively under development and can be 
+frequently updated, therefore we recommend checking the status of the
+ dependencies regularly.**
+
+## Updating your environment.
+
+To update the Plaso source code, run:
```

Comment:
remove this since origin will be a fork in most cases

Reply:
Done